Sources/Apollo/ApolloStore.swift

```diff
@@ -141,17 +141,17 @@ public final class ApolloStore {
       self.cacheKeyForObject = cacheKeyForObject
     }
     
-    public func readObject<Query: GraphQLQuery>(forQuery query: Query) throws -> JSONObject {
-      return try readObject(forSelectionSet: Query.selectionSet, withKey: rootKey(forOperation: query), variables: query.variables)
+    public func read<Query: GraphQLQuery>(query: Query) throws -> Query.Data {
```

Comment:
In Swift, argument labels are part of the method signature, so `read(query:)` is more conventional. It is in line with the standard library, and Swift actually has some pretty good [API design guidelines](https://swift.org/documentation/api-design-guidelines/).